php/libraries/NDB_BVL_Feedback.class.inc

```diff
@@ -224,7 +224,7 @@ class NDB_BVL_Feedback
      *
      * @return array
      */
-    function getFeedbackTypes()
+    static function getFeedbackTypes()
```

Comment:
public
and `php/libraries/BVL_Feedback_Panel.class.inc` is making a static and a non static call